apps/web-giddh/src/app/reports/components/columnar-report-component/columnar.report.component.scss

```diff
@@ -0,0 +1,33 @@
+form{
```

Comment:
form{
    .row{
        .col-md-2{
            &:nth-child(2),
            &:nth-child(3),
            &:nth-child(4),
            &:nth-child(5),
            &:nth-child(6){
                padding-left: 0;
            }
        }
    }
}
@media(max-width:1359px){
    form{
        .row{
            .col-md-2{
                &:nth-child(2),
                &:nth-child(3),
                &:nth-child(4),
                &:nth-child(5),
                &:nth-child(6){
                    padding-left:15px;
                }
            }
        }
    }
}

I checked this CSS unusable please remove it